src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -289,6 +306,15 @@ public void runTest() throws Exception {
             shortCircuitAnalysisMockedStatic.when(ShortCircuitAnalysis::find).thenReturn(runner);
             when(runner.getName()).thenReturn("providerTest");
 
+            //fault.id comparator
+            Comparator<FaultEmbeddable> comparatorByFaultId = comparing(FaultEmbeddable::getId);
+            //resultUuid comparator (the toString is needed because UUID comparator is not the same as the string one)
```

Comment:
`faultResultUuid` and `resultUuid` are different things, one the UUID identifying a FautlResultEntity, the other one identifying a `ShortcircuitResultEntity` maybe it's just the comment wich is confusing...

Reply:
sure it can be more explicit